org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/ITable.java

```diff
@@ -975,11 +975,11 @@ default void addUITableListener(TableListener listener, Integer... eventTypes) {
   void setTableCustomizer(ITableCustomizer c);
 
   /**
-   * Container of this table, {@link org.eclipse.scout.rt.client.ui.desktop.outline.pages.IPage IPage},
+   * Container of this table, {@link IPage IPage},
```

Comment:
Why is this simple qualified and the others below are not? Furthermore IPage only needs to be specified once if the value and name are equal.